trace/Sampling.md

```diff
@@ -12,7 +12,11 @@ The Sampling bit is always set only at the start of a Span, using a `Sampler`
 * `AlwaysSample` - sampler that makes a "yes" decision every time.
 * `NeverSample` - sampler that makes a "no" decision every time.
 * `Probability` - sampler that tries to uniformly sample traces with a given probability. When 
-applied to a child `Span` of a **sampled** parent `Span`, the child `Span` keeps the sampling decision.
+applied to a child `Span` of a **sampled** parent `Span`, the child `Span` keeps the sampling
+decision.
+* `QPS` - sampler that tries to sample traces based on QPS (e.g. 1 trace/second). When applied to a
```

Comment:
I'm not sure `QPS` is a good name for this. It's common in Google to use QPS to mean rate of anything (not just queries) but I think this is less common elsewhere in my experience, but I would be interested to hear non-Google perspectives on this. What about "fixed rate sampler"?

Reply:
In Jaeger we call them rate limiting samplers

Reply:
+1 for rate limiting - my previous suggestion of fixed rate isn't good because if the application produces fewer than this number of spans, the rate will actually be lower.

Reply:
Done.